src/mds/Server.cc

```diff
@@ -6180,6 +6183,57 @@ void Server::handle_set_vxattr(const MDRequestRef& mdr, CInode *cur)
 
     client_t exclude_ct = mdr->get_client();
     mdcache->broadcast_quota_to_client(cur, exclude_ct, true);
+  } else if (name == "ceph.quiesce.block"sv) {
```

Comment:
`ceph.quiesce.skip`?

Reply:
Closing this, continue discussion in the other thread.